python/tk_desktop/systray.py

```diff
@@ -211,7 +211,7 @@ def _pin_to_menu(self, animated=True):
 
             # setup the animation to shrink the window to the systray
             # parent the anim to self to keep it from being garbage collected
-            anim = QtCore.QPropertyAnimation(self, "geometry", self)
+            anim = QtCore.QPropertyAnimation(self, b"geometry", self)
```

Comment:
I know you have a fix for this already @jfboismenu, this is here just so that I can test my changes. I'll handle the merge conflicts when I come to merge.